plugins/obs-x264/obs-x264.c

```diff
@@ -460,6 +460,8 @@ static void update_params(struct obs_x264 *obsx264, obs_data_t *settings,
 	obsx264->params.i_height = height;
 	obsx264->params.i_fps_num = voi->fps_num;
 	obsx264->params.i_fps_den = voi->fps_den;
+	obsx264->params.i_timebase_num = voi->fps_den;
+	obsx264->params.i_timebase_den = voi->fps_num;
```

Comment:
Is this a typo or are the parameters reversed by OBS / x264?

Reply:
Timebase is the inverse of framerate, IIRC.  So 30 FPS is 1/30 s timebase for each frame.